pkg/arvo/lib/language-server/rune-snippet.hoon

```diff
@@ -0,0 +1,541 @@
+=>
+|%
+++  snippet
+  |=  [rune=tape text=tape]
+  ^-  json
+  =,  enjs:format
+  %-  pairs
+  :~  'label'^(tape rune)
+      'insertTextFormat'^(numb 2)
+      'insertText'^(tape text)
+  ==
+::
+++  runes
+  %-  malt
+  :~  :-  "|$"
```

Comment:
I'll echo @belisarius222's comment, and further note that constant-folding is trivial with `^~`:

```
> !=  (malt [1 2] [3 4] ~)
[8 [9 5.596 0 511] 9 2 10 [6 [7 [0 3] 1 1 2] [7 [0 3] 1 3 4] 7 [0 3] 1 0] 0 2]
> !=  ^~((malt [1 2] [3 4] ~))
[1 [1 2] 0 [3 4] 0 0]
```

Reply:
Bump for constant folding.

Reply:
Done